[ES] Add `osu! wiki` (#11085)
ppy/osu-wiki: wiki/osu!_wiki/es.md

## Estructura

La puerta de entrada a la osu! wiki es la [página principal](/wiki/Main_page), que se divide en varias categorías de alto nivel, cada una con una descripción general e incluye una lista de artículos importantes. Se puede acceder a los artículos que no figuran en la página principal, que suelen ser más específicos, desde otras páginas mediante enlaces en línea o mediante la función de búsqueda del sitio web.

Un artículo individual describe un solo término o concepto y ofrece una descripción general de temas adyacentes. Para ilustrar mejor el tema, se divide en subsecciones que cubren diferentes partes del término. También puede incluir capturas de pantalla relevantes u otras explicaciones visuales, o enlaces a otros artículos sobre el tema.

Hay traducciones disponibles en diferentes idiomas para algunos artículos. Los crean los propios miembros de la comunidad y la elección del idioma solo está limitada por la [lista de idiomas](/wiki/Article_styling_criteria/Formatting#configuraciones-regionales) admitida por el propio sitio web.

## Modelo

> La osu! wiki se basa en el concepto de accesibilidad: se recomiendan las ediciones y traducciones, que cualquier persona capaz y con conocimientos puede realizar. Los datos de los artículos se almacenan en un [repositorio de GitHub](https://github.com/ppy/osu-wiki) y los contribuidores los actualizan mediante el [uso de solicitudes de cambios](/wiki/osu!_wiki/Contribution_guide), que luego son revisados ​​y fusionados por los [mantenedores de la wiki](/wiki/People/osu!_wiki_maintainers).

Reviewer: ```suggestion
La osu! wiki se basa en el concepto de accesibilidad: se recomiendan las ediciones y traducciones, que cualquier persona capaz y con conocimientos puede realizar. Los datos de los artículos se almacenan en un [repositorio de GitHub](https://github.com/ppy/osu-wiki) y los contribuidores los actualizan mediante el [uso de solicitudes de cambios](/wiki/osu!_wiki/Contribution_guide), que luego son revisadas ​​y fusionadas por los [mantenedores de la wiki](/wiki/People/osu!_wiki_maintainers).
```

> Los artículos están formateados usando un [formato Markdown adaptado a GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Además, el motor del sitio web implementa características personalizadas cruciales para la wiki en su conjunto, como redireccionamientos, portada o mejoras de formato.

Reviewer: https://developer.mozilla.org/es/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN
```suggestion
Los artículos están formateados usando un [Markdown personalizado por GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Además, el motor del sitio web implementa características personalizadas cruciales para la wiki en su conjunto, como redireccionamientos, portada o mejoras de formato.
```

Los artículos están compuestos de acuerdo con los [criterios de estilo](/wiki/Article_styling_criteria) de la osu! wiki, lo que significa que están escritos en un lenguaje neutral y siguen reglas comunes de formato y diseño. Algunos artículos informativos sirven como una lista de preguntas frecuentes (FAQ) sobre el tema y usan un lenguaje informal. Las traducciones siguen el principio de paridad de contenido y cumplen con las mismas reglas de formato.

A pesar de tener el sufijo *wiki* en su nombre, la osu! wiki no satisface completamente [el concepto de Wiki](https://es.wikipedia.org/wiki/Wiki):

- Se recomienda mejorar los artículos existentes y crear otros nuevos, pero es necesario un conocimiento básico de [GitHub](https://github.com) para poder contribuir, lo que eleva el umbral necesario para ingresar. Sin embargo, la wiki de osu! proporciona una [guía de contribución](/wiki/osu!_wiki/Contribution_guide) para ayudar a los nuevos contribuidores a empezar.
- Todos los cambios en la wiki de osu! se moderan previamente y se someten a un proceso de revisión, en el que se espera que el contribuidor colabore con los revisores y responda a los comentarios.

## Licencia

> La mayor parte del contenido de la wiki osu! está bajo la licencia [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Esto significa que, en general, los materiales de la wiki se pueden utilizar de manera no comercial (consulta el [resumen rápido](https://creativecommons.org/licenses/by-nc/4.0/)). Algunos contenidos pueden estar cubiertos por diferentes licencias de terceros, que luego se citarán localmente y anularán la licencia global.

Reviewer: ```suggestion
La mayor parte del contenido de la wiki osu! está bajo la licencia [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Esto significa que, en general, los materiales de la wiki se pueden usar de manera no comercial (consulta el [resumen rápido](https://creativecommons.org/licenses/by-nc/4.0/)). Algunos contenidos pueden estar cubiertos por diferentes licencias de terceros, que luego se citarán localmente y anularán la licencia global.
```

> Ten en cuenta que esto no cubre el uso de «osu!» o marca «ppy» en cualquier software, recurso, publicidad o promoción, ya que está protegida por la ley de marcas. Si necesitas autorización para el uso de estos términos, [contáctenos](mailto:contact@ppy.sh).

Reviewer: ```suggestion
Ten en cuenta que esto no cubre el uso de «osu!» o marca «ppy» en cualquier software, recurso, publicidad o promoción, ya que está protegida por la ley de marcas. Si necesitas autorización para el uso de estos términos, [contáctanos](mailto:contact@ppy.sh).
```
